media-libs/imlib2/imlib2-1.4.9.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="An advanced replacement library for libraries like libXpm"
+DESCRIPTION="A library that does image file loading and \
+saving as well as rendering, manipulation, arbitrary polygon support, etc.
+
+It does ALL of these operations FAST. Imlib2 also tries to be highly \
+intelligent about doing them, so writing naive programs can be done \
+easily, without sacrificing speed.
+
+This is a complete rewrite over the Imlib 1.x series. The architecture is \
+more modular, simple, and flexible. See index.html in the doc/ directory \
+for more information."
+HOMEPAGE="https://www.enlightenment.org/"
+SOURCE_URI="http://downloads.sourceforge.net/project/enlightenment/imlib2-src/$portVersion/imlib2-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7d2864972801823ce44ca8d5584a67a88f0e54e2bf47fa8cf4a514317b4f0021"
+REVISION="1"
+LICENSE="IMLIB2"
+COPYRIGHT="2002 Project Enlightenment"
```

Comment:
COPYRIGHT should go right after HOMEPAGE, as recommended in the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).
LICENSE should go right after COPYRIGHT.